enterprise_access/apps/api_client/constants.py

```diff
@@ -0,0 +1,16 @@
+"""
+Constants for API client
+"""
+import backoff
+from requests.exceptions import ConnectionError as RequestsConnectionError
+from requests.exceptions import HTTPError
+from requests.exceptions import Timeout as RequestsTimeoutError
+
+# wait_gen references the field in the backoff decorator representing
+# the backoff strategy for retries
+wait_gen = backoff.expo
```

Comment:
What does wait_gen mean? Could we make this something more descriptive? Perhaps this already matches other implementations and I just just let it go?

Reply:
I sourced the name from the [arguments](https://github.com/litl/backoff/blob/d82b23c42d7a7e2402903e71e7a7f03014a00076/backoff/_decorator.py#L123-L137) of `on_exception` but it makes sense that the naming should be more description. Updated 👍🏽